core/modules/saver-handler.js

```diff
@@ -95,13 +95,15 @@ function SaverHandler(options) {
 	if($tw.browser) {
 		$tw.rootWidget.addEventListener("tm-save-wiki",function(event) {
 			self.saveWiki({
+				wiki: event.widget.wiki,
```

Comment:
I find it interesting that the core saver-handler.js required a change. Could you explain?

Reply:
Hi @btheado I was a bit surprised too: it is because the message that we use to save the wiki was previously hardwired to use the main wiki as the source for the tiddlers and template, and so wasn't able to save the payload tiddlers. The new code uses the wiki associated with the wiki that sent the message, which should be backwards compatible.